### ISSUE TITLE
BBP setting not saved

### ISSUE THREAD

**Issue by ChildishGiant** [nesbox/TIC-80#1559]
It would be nice if the bbp that was chosen in the editor was saved so I don't have to go through all the menus each time I reload the cart and want to use the sprite editor etc. This could either be saved in the cart or the editor itself could remember based on what cart you loaded.

**Comment by nesbox** [comment]
We can go even deeper and save all the state of the editors, the console commands history, etc.
Thank you.

**Comment by joshgoebel** [comment]
> We can go even deeper and save all the state of the editors, the console commands history, etc.

Where?  Into each individual cartridge?  I like the idea, but I'm not sure *everything* would be the corrects scope, one could imagine the console session continuing for a while and working with several different cartridges... though perhaps all console activity since the cartridge was opened (until another is opened) is fair game?

**Comment by nesbox** [comment]
No, not for every cart, I think it could be a file with the current console state that is restored on every run.

**Comment by joshgoebel** [comment]
That would surely be useful, but it wouldn't solve this - at least not in the specific problem I see presented:

>  This could either be saved in the cart or the editor itself could remember **based on what cart you loaded.** [emphasis added]

One images that someone regularly edits TWO (or more) cartridges... one with 1bpp sprites and the other with 4bpp sprites... I wouldn't want the editor to remember the last setting, I'd want it to -smartly- realizing that with one cartridge I last used 1bpp while with the other cartridge I last used 4bpp.
